src/Conversion/ONNXToTOSA/ONNXToTOSACommon.hpp

```diff
@@ -149,7 +149,7 @@ namespace onnx_mlir {
 inline bool isTOSASignedInt(mlir::Type type) {
   mlir::IntegerType intType = type.dyn_cast<mlir::IntegerType>();
   std::set<unsigned> intWidth{8, 16, 32, 48, 64};
-  return intType && intType.isSignless() &&
+  return intType &&
```

Comment:
Why was this necessary? If the two new pattern have a special case, maybe we need to add a second type converter or check manually. I don't feel good just loosening the constraints here